keyboards/handwired/jotpad16/config.h

```diff
@@ -16,13 +16,13 @@
 
 /* pro_micro pin-out */
 #define MATRIX_ROW_PINS { B6, B2, D2, D3 }
-#define MATRIX_COL_PINS { B5, B4, B3, B1 }
+#define MATRIX_COL_PINS { E6, D7, B3, B1 }
 #define UNUSED_PINS
 
 /* leds */
-#define QMK_LED E6
```

Comment:
You're removing an LED from the QMK bootloader config. Are you sure that's what you mean to do there?  

You can use it for both, if that matters.

Reply:
ah ok, i use it for both in that case, TY!